net/wireguard/Makefile

```diff
@@ -36,6 +36,7 @@ define Package/wireguard/Default
   SUBMENU:=VPN
   URL:=https://www.wireguard.io
   MAINTAINER:=Baptiste Jonglez <openwrt@bitsofnetworks.org>, \
+	      Kevin Darbyshire-Bryant <ldir@darbyshire-bryant.me.uk> \
```

Comment:
Looks like there's a mixed tabs and spaces thing going on here, when there should only be spaces.

Reply:
fixed!